kafka_versions.go

```diff
@@ -0,0 +1,10 @@
+package sarama
+
+var LatestStable, _ = NewVersion("0.9.0.1")
+var V0_10_0, _ = NewVersion("0.10.0.0")
+var V0_9_0_1 = LatestStable
```

Comment:
I'd prefer defining this with a literal `var V0_9_0_1 = NewVersion("0.9.0.1")` and then define the latest stable at the bottom with `var LatestStable = V0_9_0_1`.